sql/core/src/test/scala/org/apache/spark/sql/execution/SparkSqlParserSuite.scala

```diff
@@ -301,6 +302,185 @@ class SparkSqlParserSuite extends AnalysisTest {
         List.empty, List.empty, None, None, false)))
   }
 
+  test("SPARK-28227: script transform with row format delimit with aggregation") {
+    assertEqual(
+      """
+        |SELECT TRANSFORM(a, sum(b), max(c))
+        |  ROW FORMAT DELIMITED
+        |  FIELDS TERMINATED BY ','
+        |  COLLECTION ITEMS TERMINATED BY '#'
+        |  MAP KEYS TERMINATED BY '@'
+        |  LINES TERMINATED BY '\n'
+        |  NULL DEFINED AS 'null'
+        |  USING 'cat' AS (a, b, c)
+        |  ROW FORMAT DELIMITED
+        |  FIELDS TERMINATED BY ','
+        |  COLLECTION ITEMS TERMINATED BY '#'
+        |  MAP KEYS TERMINATED BY '@'
+        |  LINES TERMINATED BY '\n'
+        |  NULL DEFINED AS 'NULL'
```

Comment:
I feel the part above is redundant because we already have the test for it, I think: https://github.com/apache/spark/blob/14c2edae7e8e02e18a24862a6c113b02719d4785/sql/core/src/test/scala/org/apache/spark/sql/execution/SparkSqlParserSuite.scala#L328-L336

So, how about removing it from these tests?